app/src/organisms/ProtocolSetupParameters/__tests__/ChooseEnum.test.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react'
+import { it, describe, beforeEach, vi, expect } from 'vitest'
+import { fireEvent, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/vitest'
+import { COLORS } from '@opentrons/components'
+import { renderWithProviders } from '../../../__testing-utils__'
+import { i18n } from '../../../i18n'
+import { ChooseEnum } from '../ChooseEnum'
+
+vi.mocked('../../ToasterOven')
+const render = (props: React.ComponentProps<typeof ChooseEnum>) => {
+  return renderWithProviders(<ChooseEnum {...props} />, {
+    i18nInstance: i18n,
+  })
+}
+describe('ChooseEnum', () => {
+  let props: React.ComponentProps<typeof ChooseEnum>
+
+  beforeEach(() => {
+    props = {
+      handleGoBack: vi.fn(),
+      parameter: {
+        displayName: 'Dry Run',
+        variableName: 'DRYRUN',
```

Comment:
nit
probably snake case would be normal in python. 
```suggestion
        variableName: 'dry_run',
```